supplier/migrations/0001_initial.py

```diff
@@ -30,6 +35,9 @@ class Migration(migrations.Migration):
                 ('business_name', models.CharField(help_text='Business Name',
                                                    max_length=32,
                                                    validators=[django.core.validators.MinLengthValidator(6)])),
+                ('supplier_account', models.OneToOneField(null=True,
```

Comment:
Was this auto-generated?

Reply:
I have updated the migration with deleting the old ones and creating the new ones and yes it is from make migrations

Reply:
**Never ever** modify a migration manually. this should be reverted. keep the old migration file, then run make migrations as usual.

Reply:
I did not modify it manually,i just removed the old one and makemigrations again so it updates few things that was on the old migrations and add some new ones

Reply:
What i meant by modify is deleting old one and creating new one so that new one has somethings that was not in the old one,i know that i should not update a migration file and there is no need to do that since there is built in something that does the migration work.

Reply:
removing an old one == modifying it manually. migrations don't work this way, but they're building the changes on top of each other.

Reply:
I know if you update something and makemigration then it will modify it on top of the last one but if you delete all,it will modify all automatically and use the code in one file

Reply:
using makemigrations and modifying on top is the right approach. think of a scenario when you already had a DB active and now you made a change to the schema - the migration needs to know what it had, and what it needs to have now, in order to perform the change successfully.

Reply:
I know but the results will be the same results but in the modifying on top will let us know what did happend and what fields have been modified and more,but at the moment we have only one migrations that is correct and i think i can not go back to the old ones

Reply:
why won't you be able to go back? just reset the migration file specifically to the one in `main`, then run makemigrations as you should've done in the beginning. this fix will take less time than the discussion we had. :) 
`git checkout origin/master [filename-to-reset]`

Reply:
Fixed!